deployments/helm-chart/crds/ap-logconf.yaml

```diff
@@ -5,16 +5,14 @@ metadata:
     controller-gen.kubebuilder.io/version: v0.2.5
   creationTimestamp: null
   name: aplogconfs.appprotect.f5.com
-  labels:
-    app.kubernetes.io/name: "nginx-ingress"
```

Comment:
Although while these labels are proposed to be removed in `https://github.com/nginxinc/kubernetes-ingress/pull/1073`

In this PR these should be re-added in my opinion.